scripts/install

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh -e
+
+[[ $1 = "-p" ]] && PYTHON=$2 || PYTHON="python"
```

Comment:
I'd suggest we don't bother with this - just use "python".
If someone wants to install with some other version other than their system default then they can run virtualenv manually.
It's important that we keep the script simple so that it's as obvious as possible what it does.

Reply:
I hear your concern, but:

- I believe on macOS `python` generally refers to the system's 2.7 version, and I'd bet some Linux distros still do the same.
- It seems many Windows installs expose the Python interpreter as `py`.

If we just hardcode the `python` executable, the script is unusable for a lot of users which IMO defeats its purpose.

If the difficulty of understanding what this lines does is the problem, I can add a thoughtful comment.

Reply:
Okay.
Thinking about it twice, the default should be `python3` in any case.

Reply:
Oops, that's actually what I indicated in `CONTRIBUTING.md` but not in the script. Thanks, that's fixed.